### DISCUSSION TITLE
Video Embedding on NgxEditors, image upload from local system

### DISCUSSION THREAD

**Discussion by pramodpadmanabhi** [sibiraj-s/ngx-editor#506] [General]
How to embed video link? is it already available by default or do we have to create custom menu
How to upload image from local system , the custom menu link which shared previously is broken

**Comment by sibiraj-s** [comment]
Here the link for custom menu. https://sibiraj-s.github.io/ngx-editor/menu/#custom-menu. i18n redirects needs setup. Will fix that.

No its not supported by default, you will need to render custom nodeview for that.

You can follow the image component for reference
- https://github.com/sibiraj-s/ngx-editor/blob/master/projects/ngx-editor/schema/nodes.ts#L205 
this is just schema, you can browse through the codebase for additional details. It should be straight forward.

[a comment by an automated account is not shown]
